tests/conftest.py

```diff
@@ -2592,3 +2592,20 @@ def ns_resources_and_connections_cleanup():
     request.addfinalizer(ns_resources_and_connections_cleanup)
 
     return _create_ns_resources
+
+
+@pytest.fixture(scope="session", autouse=True)
+def collect_logs_fixture(request):
+    """
+    This fixture collects ocs logs after tier execution and this will allow
+    to see the cluster's status after the execution on all execution status options.
+    """
+    def finalizer():
+        """
+        Tracking both logs separately reduce changes of collision
+        """
+        if not config.RUN['cli_params'].get('deploy') and not config.RUN['cli_params'].get('destroy'):
```

Comment:
+1

Reply:
After reviewing logs provided by @jimdevops19 I see that this may not be a problem. The hook collects must gathers right after each failed test and this new fixture collects logs after test run finishes. It might be useful to have new must gathers right after all tests finish even if we already have some must gathers collected from failed test cases. @petr-balogh @clacroix12 What do you think?

Reply:
That's the case here (why we have the issue opened) to collect at least once the must gather after all tests finished.  Or I didn't get you @fbalak